src/style/colors.ts

```diff
@@ -8,17 +8,21 @@ import {colorsPalette, themeColors} from './colorsPalette';
 import ColorName from './colorName';
 
 type Schemes = {light: {[key: string]: string}; dark: {[key: string]: string}};
+type CurrentScheme = 'default' | 'light' | 'dark'
```

Comment:
I suggest renaming the type to `Scheme` or `SchemeType`. 
`CurrentScheme` is the name of the instance